### ISSUE TITLE
Galactic and Space Sciences

### ISSUE THREAD

**Issue by jeekjeek44** [nasa/apod-api#55]
Can we, like the rest of the inhabitants of the planet Earth, who love science, astronomy and space, to absorb them in universities in order to serve the interest of mankind and discover broader horizons in development and knowledge .. ??

**Comment by iyinolu** [comment]
Absolutely! I've been thinking about this for some time now. What solutions do you propose though?
